<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertDialog</class>
 <widget class="QDialog" name="InsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert breakpoints</string>
  </property>
  <property name="windowIcon">
   <iconset theme="list-add">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="breakGroup">
     <property name="title">
      <string>Insert breakpoints on existing points</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="breaksLayout">
      <item>
       <widget class="QCheckBox" name="endsCheck">
        <property name="text">
         <string>Insert breakpoint at line end</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="breaksCheck">
        <property name="text">
         <string>Insert breakpoint on segment break</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="intersectCheck">
        <property name="text">
         <string>Insert breakpoint at lines intersection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="symbolCheck">
        <property name="text">
         <string>Insert breakpoint on point object</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="segmentGroup">
     <property name="title">
      <string>Insert segments in surface center</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="segmentsLayout">
      <item>
       <widget class="QCheckBox" name="outsideCheck">
        <property name="text">
         <string>Insert only near boundary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hideCheck">
        <property name="text">
         <string>Hide inserted segments</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>358</width>
       <height>150</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QDoubleSpinBox" name="radiusSpin">
     <property name="specialValueText">
      <string>Match exactly</string>
     </property>
     <property name="prefix">
      <string>Search in </string>
     </property>
     <property name="suffix">
      <string> m radius</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>endsCheck</tabstop>
  <tabstop>breaksCheck</tabstop>
  <tabstop>intersectCheck</tabstop>
  <tabstop>symbolCheck</tabstop>
  <tabstop>radiusSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InsertDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InsertDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endsCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>InsertDialog</receiver>
   <slot>insertParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breaksCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>InsertDialog</receiver>
   <slot>insertParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>intersectCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>InsertDialog</receiver>
   <slot>insertParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>symbolCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>InsertDialog</receiver>
   <slot>insertParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>breakGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>InsertDialog</receiver>
   <slot>insertParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>642</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>segmentGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>InsertDialog</receiver>
   <slot>insertParamsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>insertParamsChanged()</slot>
 </slots>
</ui>
